poe-tasks/gradle-connector-tasks.toml

```diff
@@ -27,18 +27,24 @@
 
 [tasks]
 
-# Universal Connector Tasks
-# These tasks are available in all connectors, regardless of the language.
+get-language = "echo 'java'" # Use with -qq to get just the language name
+fetch-secrets = "airbyte-cdk secrets fetch"
+
+install = [
+  "gradle-warmup",
+  "install-cdk-cli",
+]
```

Comment:
Pre-warming gradle should make subsequent steps faster and their logs slightly cleaner. Also, if there's a failure to compile the gradle tasks, it will fail here, rather than later when we expect failures to mean a test failure.

Reply:
We don't have to get into this here - I was thinking of using `install` to reference gradle `assemble` which would build the java tar and docker image. Alternatively we can add an actual `build` command for jvm connectors only - straightforward now that you've split the GHA files.